lib/app/features/feed/views/components/feed_item/feed_item_footer/entity_likes_button.dart

```diff
@@ -13,9 +13,14 @@ import 'package:ion/app/utils/num.dart';
 import 'package:ion/generated/assets.gen.dart';
 
 class EntityLikesButton extends ConsumerWidget {
-  const EntityLikesButton({required this.eventReference, super.key});
+  const EntityLikesButton({
```

Comment:
Maybe we could rename `EntityLikesButton` and `EntityRepliesButton` to `ReelLikesButton` and `ReelRepliesButton` to clarify their purpose and better reflect the domain-specific concept of '`Reels`'? This might make the code more intuitive and easier to read.